pkg/apis/autoscaling/register.go

```diff
@@ -26,11 +26,38 @@ const (
 	//   autoscaling.knative.dev/class: foo
 	// This uses a different domain because unlike the resource, it is user-facing.
 	ClassAnnotationKey = GroupName + "/class"
-
+	// KPA is Knative Horizontal Pod Autoscaler
+	KPA = "kpa"
+	// HPA is Kubernetes Horizontal Pod Autoscaler
+	HPA = "hpa"
+
+	// MinScaleAnnotationKey is the annotation to specify the minimum number of Pods
+	// the PodAutoscaler should provision. For example,
+	//   autoscaling.knative.dev/minScale: "1"
 	MinScaleAnnotationKey = GroupName + "/minScale"
+	// MinScaleAnnotationKey is the annotation to specify the maximum number of Pods
+	// the PodAutoscaler should provision. For example,
+	//   autoscaling.knative.dev/maxScale: "10"
 	MaxScaleAnnotationKey = GroupName + "/maxScale"
 
+	// MetricAnnotationKey is the annotation to specify what metric the PodAutoscaler
+	// should be scaled on. For example,
+	//   autoscaling.knative.dev/metric: cpu
+	MetricAnnotationKey = GroupName + "/metric"
+	// Concurrency is the number of requests in-flight at any given time.
+	Concurrency = "concurrency"
+	// OPS is the number of requests (operations) arriving per second.
+	OPS = "ops"
```

Comment:
Is this dead code now, given it's not implemented yet? Shall we leave it out until it is?